src/test/java/com/hotels/beeju/extensions/HiveMetaStoreJUnitExtensionTest.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (C) 2015-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju.extensions;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.hadoop.hive.metastore.api.AlreadyExistsException;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class HiveMetaStoreJUnitExtensionTest {
+
+  @RegisterExtension
+  HiveMetaStoreJUnitExtension hiveDefaultName = new HiveMetaStoreJUnitExtension();
```

Comment:
Can these be made private?

Reply:
According to the J5 docs: 
"@RegisterExtension fields must not be private or null (at evaluation time) but may be either static or non-static." (https://junit.org/junit5/docs/current/user-guide/#extensions-registration-programmatic) although it doesn't tell me why :((

Reply:
OK, that's fine, so these have to be left with this level of access then, good to know.